lib/ui/mail_view_indv.dart

```diff
@@ -1,24 +1,71 @@
+import 'dart:convert';
+import 'package:enough_mail/codecs.dart';
 import 'package:flutter/material.dart';
-import 'package:flutter/widgets.dart';
+import 'package:global_configuration/global_configuration.dart';
 import 'package:intl/intl.dart';
 import 'package:summer2022/models/MailPiece.dart';
 import 'package:summer2022/ui/bottom_app_bar.dart';
+import '../models/Digest.dart';
+import '../services/mail_retrieveByMailPiece.dart';
 
-class MailPieceViewWidget extends StatelessWidget{
+class MailPieceViewWidget extends StatefulWidget{
+
+  final MailPiece mailPiece;
+
+  const MailPieceViewWidget({Key? key, required this.mailPiece}) : super(key: key);
+
+  @override
+  MailPieceViewWidgetState createState() => MailPieceViewWidgetState(mailPiece);
+}
+
+  GlobalConfiguration cfg = GlobalConfiguration();
+
+  class MailPieceViewWidgetState extends State<MailPieceViewWidget> {
+  GlobalConfiguration cfg = GlobalConfiguration();
 
   final FontWeight _commonFontWeight = FontWeight.w500;
   final double _commonFontSize = 30;
   final Color _buttonColor = Color.fromRGBO(51, 51, 102, 1.0);
+  final mailPiece;
+  late Digest digest;
+  late Image mailImage = Image.asset('assets/mail.test.02.png');
 
-  final MailPiece mailPiece;
-  MailPieceViewWidget({required this.mailPiece});
+  MailPieceViewWidgetState(this.mailPiece);
+
+  @override
+  void initState() {
+    super.initState();
+    _getMailPieceEmail();
+  }
+
+  Future<void> _getMailPieceEmail() async {
+      MailPieceEmailFetcher mpef1 = await MailPieceEmailFetcher(mailPiece);
```

Comment:
At this point we should just be  using the MailPiece data to populate the form. The only extra thing needed I would think is to fetch the original image since we only store the OCR text.

Reply:
Exactly, talking with Mike, he specifically wanted me to fetch the email picture from the mail id.  I'm sure it can be simplified but I made it with a constructor.